ocpmodels/trainers/forces_trainer.py

```diff
@@ -380,7 +321,7 @@ def predict(
         if results_file is not None:
```

Comment:
Do we want to move the `results_file` writing (lines 321-355) to the `base_trainer` as well? Almost identical between energy and force trainer. Only difference is the extra force lines at 331 and 350?

Reply:
Good point. I've updated the PR with this.

Reply:
Awesome, looks good - thanks!